src/vivarium_csu_ltbi/components/risk.py

```diff
@@ -0,0 +1,73 @@
+from vivarium_public_health.utilities import EntityString, TargetString
+from vivarium_public_health.risks.data_transformations import (get_relative_risk_data,
+                                                               get_population_attributable_fraction_data)
+
+
+class HHTBCorrelatedRiskEffect:
+    """A custom risk effect that targets prevalence and birth
+    prevalence of LTBI and adjusts it to correlate it with household
+    tuberculosis"""
+
+    configuration_defaults = {
+        'effect_of_risk_on_target': {
+            'measure': {}
+        }
+    }
+
+    def __init__(self, target):  # sequela.ltbi_susceptible_hiv, sequela.ltbi_positive_hiv
+        self.risk = EntityString('risk_factor.household_tuberculosis')
+        self.target = TargetString(target)
+        self.configuration_defaults = {
+            f'effect_of_{self.risk.name}_on_{self.target.name}': {
+                self.target.measure:
+                    HHTBCorrelatedRiskEffect.configuration_defaults['effect_of_risk_on_target']['measure']
+            }
+        }
+
+    @property
+    def name(self):
+        return f'risk_effect.household_tuberculosis.{self.target.name}.{self.target.measure}'
+
+    def setup(self, builder):
+        self.randomness = builder.randomness.get_stream(
+            f'effect_of_{self.risk.name}_on_{self.target.name}.{self.target.measure}'
+        )
+
+        self.relative_risk = self._get_relative_risk_date(builder)
+        self.exposed_exposure_probability = self._get_exposed_exposure_probability_data(builder)
+        self.exposure = builder.value.get_value(f'{self.risk.name}.exposure')
+        self.population_attributable_fraction = self._get_population_attributable_fraction_data(builder)
+
+        builder.value.register_value_modifier(f'{self.target.name}.{self.target.measure}',
+                                              modifier=self.adjust_target,
+                                              requires_values=[f'{self.risk.name}.exposure'],
+                                              requires_columns=['age', 'sex'])
+
+    def _get_relative_risk_date(self, builder):
```

Comment:
Is this intended or a typo: 
_get_relative_risk_date
or
_get_relative_risk_data

Reply:
not intended :sweat_smile: